src/assets/toolkit/styles/components/calendar-date.css

```diff
@@ -29,6 +29,7 @@
   min-height: var(--CalendarDate-min-height);
   min-width: var(--CalendarDate-min-width);
   text-align: center;
+  overflow: hidden; /* round top corners of inner header */
```

Comment:
This is great, but please use [our typical comment structure](https://github.com/cloudfour/guides/tree/master/css#comments).